requirements.txt

```diff
@@ -0,0 +1,8 @@
+dandi
+pandas
+streamlit
+st-theme
+pillow
+plotly
+numpy
+s3fs
```

Comment:
Do you use this import anywhere?

Reply:
`s3fs` is required for reading data from S3 with Pandas.  I can't recall exactly, but I believe `pillow` is needed for `px.imshow` or `st.plotly_chart`.  (And `plotly` and `numpy` are used explicitly in `home.py`.)

Reply:
Actually, `pillow` is need for the favicon.